crates/biome_js_analyze/tests/specs/nursery/noLeakedRender/invalid.jsx

```diff
@@ -0,0 +1,70 @@
+// Invalid cases with default options (both 'coerce' and 'ternary' strategies)
```

Comment:
```suggestion
// should generate diagnostics
// Invalid cases with default options (both 'coerce' and 'ternary' strategies)
```

FYI If you accept, this will fail the snapshot

Reply:
I have removed my old comment and replaced with `should generate diagnostics`